integrated_channels/blackboard/client.py

```diff
@@ -57,17 +59,40 @@ def create_content_metadata(self, serialized_data):
 
         external_id = channel_metadata_item.get('externalId')
 
+        # Retrieve and pop the data not needed for course creation and necessary for content customization
+        enrollment_link_metadata = channel_metadata_item.pop('enrollment_metadata')
```

Comment:
wrt error handling, do we still want to allow course creation if something happens in the process of adding this content (keyError or any other error?). Similarly, what what a failure once course is created, but the adding of this metadata/customizatoin fails? How do we want to handle that? I think since course was created, it should still be reported as successful (otherwise will integrated channels system try to create course again hi the future?)

let's maybe just think over this aspect a bit...